app/pom.xml

```diff
@@ -98,6 +99,43 @@
                     </webResources>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>0.0.20</version>
```

Comment:
I think you will need to use latest version to make travis ci happy (0.0.23 or 0.0.24 I believe)